src/OutputReaders/field_dataset.jl

```diff
@@ -80,3 +83,114 @@ function Base.show(io::IO, fds::FieldDataset)
 
     return print(io, s)
 end
+
+"""
+    FieldDataset(grid, field_names, times;
+                 backend=OnDisk(),
+                 path=nothing,
+                 locations=nothing,
+                 indices=nothing)
+
+Returns a `FieldDataset` containing a new `FieldTimeSeries` for each key in `field_names`
+on `grid` at `times`.
+
+Keyword arguments
+=================
+- `backend`: backend, `InMemory(indices=Colon())` or `OnDisk()`
+
+- `path`: path to data for `backend = OnDisk()`
+
+- `location`: `Tuple` of location specifications, defaults to 
+               (Center, Center, Center) for each field`
+
+- `indices`: `Tuple` of spatial indices, defaults to (:, :, :) for each field
+
+- `boundary_conditions`: `Tuple` of boundary conditions
+
+- `metadata`: `Dict` containing metadata entries
+"""
+function FieldDataset(grid, fields::NTuple{N, Symbol}, times;
+    backend=OnDisk(),
+    path=nothing,
+    location=NamedTuple(),
+    indices=NamedTuple(),
+    boundary_conditions=NamedTuple(),
+    metadata=Dict(),
+    reader_kw=NamedTuple()) where {N}
```

Comment:
possibly, can use `fts_kw...` as a placeholder for all the kwargs that get passed without modification/inspection into `FieldTimeSeries`.